reloj_checador_team03/config/database.yml

```diff
@@ -24,8 +24,8 @@ default: &default
 development:
   <<: *default
   database: reloj_checador_team03_development
-  username: miguel
-  password: tscom2020
+  username: fego
+  password: kurosu
   host: localhost
   port: 5432
 
```

Comment:
It's a good practice to not track this file since we are able to config as we want locally.

I'd suggest moving this file to `database.yml.example` as a placeholder of what is needed in order to make the app run locally.